src/libraries/System.Private.CoreLib/src/System/Environment.Browser.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System
+{
+    public static partial class Environment
+    {
+        // Emscripten VFS mounts at / and is the only drive
+        public static string[] GetLogicalDrives() => new string[] { "/" };
+
+        // In the mono runtime, this maps to gethostname, which returns 'emscripten'.
+        // Returning the value here allows us to exclude more of the runtime.
+        public static string MachineName
+        {
+            get { return "emscripten"; }
```

Comment:
Nit: Inconsistent formatting. All properties in this file can just use the single-line `=>`

Reply:
I need to get up w/ the times ;-).